quirks/password-rules.json

```
@@ -65,6 +65,9 @@
     "battle.net": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower, upper; allowed: digit, special;"
     },
+    "benefitslogin.discoverybenefits.com": {
+        "password-rules": "minlength: 10; required: upper; required: digit; required: special; allowed: [!#$%&*?@];"
```

`required: special; allowed: [!#$%&*?@];` should be compacted down to `required: [!#$%&*?@];`. As it is currently written, this rule allows for any special character in the `special` character class (since the characters allowed by a password rule are the union of the `required` and `allowed` character classes), which isn't quite what we want here 🙂.

After that, we can add `allowed: lower;` to this rule, since the site allows for lower-case characters based on the screenshot.

---

Thanks, @Cldfire! Does this look better?

```suggestion
        "password-rules": "minlength: 10; required: upper; required: digit; required: [!#$%&*?@]; allowed: lower;"
```

---

Yep, that looks great 😄